### ISSUE TITLE
cameraPoint and targetPoint get swapped in example

### ISSUE THREAD

**Issue by mmmenno** [nypl-spacetime/Leaflet.GeotagPhoto#8]
In https://github.com/nypl-spacetime/Leaflet.GeotagPhoto/blob/master/examples/camera.html targetPoint and cameraPoint get swapped, so 

```js
var points = {
        type: 'Feature',
        properties: {
          angle: 20
        },
        geometry: {
          type: 'GeometryCollection',
          geometries: [
            {
              type: 'Point',
              coordinates: cameraPoint
            },
            {
              type: 'Point',
              coordinates: targetPoint
            }
          ]
        }
      }
```
should be

```js
var points = {
        type: 'Feature',
        properties: {
          angle: 20
        },
        geometry: {
          type: 'GeometryCollection',
          geometries: [
            {
              type: 'Point',
              coordinates: targetPoint
            },
            {
              type: 'Point',
              coordinates: cameraPoint
            }
          ]
        }
      }
```

**Comment by bertspaan** [comment]
Thanks! This does not just happen in example, but in every initialization of Camera tool, see https://github.com/nypl-spacetime/Leaflet.GeotagPhoto/blob/master/src/Leaflet.GeotagPhoto.Camera.js#L136-L145

**Comment by bertspaan** [comment]
Fixed in b48d8f8b765a39d8a675436c1689ef5f0e6f4a66.
